web/vocab/TERMS.md

```diff
@@ -0,0 +1,109 @@
+
+## Sense
+- One of the five common human senses: Sight, Sound, Smell, Touch, Taste.  See http://id.worldcat.org/fast/1112562
+
+## Practice
+- An action being performed by individual(s) based on religious guidance.  See http://id.worldcat.org/fast/1149637
+
+## Object
+- Observable physical matter. See http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#PhysicalResource
+
+## RelatedSenses
+- Aggregates the various Senses under an ExperienceEvent.
+
+## RelatedPractices
+- Aggregates the various Practices under an ExperienceEvent.
+
+## RelatedObjects
+- Aggregates the various Objects under an ExperienceEvent.  See http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#hasArtefactRelatedPhysicalResource
+
+## Sight
+- A Person's Sense of Sight, noting the person and related phenomenon causing the sense.  See http://id.worldcat.org/fast/1167852  
+
+## Sound
+- A Person's Sense of Sound, noting the person and related phenomenon causing the sense. See http://id.worldcat.org/fast/1127099
+
+## Feel
+- A Person's Sense of Feel, noting the person and related phenomenon causing the sense. See http://id.worldcat.org/fast/1153089
+
+## Smell
+- A Person's Sense of Smell, noting the person and related phenomenon causing the sense.  See http://id.worldcat.org/fast/1121562
+
+## Taste
+- A Person's Sense of Taste, noting the person and related phenomenon causing the sense. See http://id.worldcat.org/fast/1143475
+
+## Singing
+- The act of singing being performed by a Person or Persons.  See http://id.worldcat.org/fast/1119311
+
+## Sitting
+- The act of sitting being performed by a Person or Persons. See http://id.worldcat.org/fast/1073252
+
+## Kneeling
+- The act of kneeling being performed by a Person or Persons. See http://id.worldcat.org/fast/1073252
+
+## Standing
+- The act of standing being performed by a Person or Persons. See http://id.worldcat.org/fast/1073252
+
+## Traveling
+- The act of traveling being performed by a Person or Persons.  See http://id.worldcat.org/fast/1155558
+
+## Dancing
+- The act of dancing being performed by a Person or Persons.  See http://id.worldcat.org/fast/887402
+
+## Working
+- The act of work to produce some predefined outcome being performed by a Person or Persons.
+
+## Cooking
+- The act of preparing Food. See http://id.worldcat.org/fast/1754966 
+
+## ClotheAction
+- The act of placing an article of Clothing onto a Person or Object.  See http://id.worldcat.org/fast/864680
+
+## Clothing
+- Garments worn by a Person that act to cover parts of the body.  See http://id.worldcat.org/fast/864680
+
+## Food
+- Consumable material for human consumption.  See http://id.worldcat.org/fast/930458, http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#Food
+
+## Natural
+- Comes from the Universe without (known) interaction from outside agents.  See http://id.worldcat.org/fast/1034403
```

Comment:
"Agents" is too broad. Maybe "without apparent synthetic design, processing, or manufacture"

Reply:
lol